Pipfile

```diff
@@ -1,14 +1,24 @@
 [[source]]
-url = "https://pypi.python.org/simple"
-name = "pypi"
+url = "https://pypi.org/simple"
 verify_ssl = true
+name = "pypi"
 
 [packages]
-google-cloud-pubsub = "*"
-notifications-python-client = "*"
-structlog = "*"
-asyncio = "*"
+flask = "*"
+pyjwt = "*"
+requests = "*"
+
+[dev-packages]
+black = "20.8b1"
+mypy = "*"
 pylint = "*"
+pytest = "*"
+flake8 = "*"
+pylint-quotes = "*"
+responses = "*"
+pytest-cov = "*"
+functions-framework = "*"
+python-dotenv = "*"
```

Comment:
dotenv can be removed now?

Reply:
hmm - I'm using it for local dev... can we keep it given it's only a dev dependency?